source/particles/particle_handler.inst.in

```diff
@@ -19,6 +19,21 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
 #if deal_II_dimension <= deal_II_space_dimension
     namespace Particles
     \{
+      namespace internal
+      \{
+        template std::vector<char>
+        pack_particles(
```

Comment:
Do you actually need this? These are functions internal to the .cc file. We shouldn't need explicit instantiations. In fact, I think that these functions could also just be in an anonymous namespace.